docs/preprocess_dataset.md

```diff
@@ -677,23 +677,25 @@ h36m_p1=dict(
 
 For data preparation of [Human3.6M](http://vision.imar.ro/human3.6m/description.php) for HMR and SPIN training, we use the [MoShed](https://mosh.is.tue.mpg.de/) data provided in [HMR](https://github.com/akanazawa/hmr) for training. However, due to license limitations, we are not allowed to redistribute the data. Even if you do not have access to these parameters, you can still generate the preprocessed h36m npz file without mosh parameters using our [converter](https://github.com/open-mmlab/mmhuman3d/tree/main/mmhuman3d/data/data_converters/h36m.py).
 
-To do so, modify the `h36m_p1` config in [DATASET_CONFIG](https://github.com/open-mmlab/mmhuman3d/blob/main/tools/convert_datasets.py):
+You will need to extract images from raw videos for training. To do so, modify the `h36m_p1` config in [DATASET_CONFIG](https://github.com/open-mmlab/mmhuman3d/blob/main/tools/convert_datasets.py):
```

Comment:
I think it would be great to also warn users about the preprocessing time can be long.

Reply:
added in [e7508ad](https://github.com/open-mmlab/mmhuman3d/pull/143/commits/e7508ad999b2657d1b343980111abe5f07486f0d)